src/firehose.rs

```diff
@@ -0,0 +1,3 @@
+//! Amazon Kinesis Firehose
+
```

Comment:
Clippy is throwing warnings regarding the fact that all of the enum variants for the `Error` enums have the `Exception` postfix (which I'm not particularly a fan of), but since you've decided to generate them that way, could you please add the following here to suppress these warnings:

```rust
#![cfg_attr(feature = "nightly-testing", allow(enum_variant_names))]
```